crypto/crypto.ts

```diff
@@ -328,6 +349,13 @@ const webCryptoDigestAlgorithms = [
 export type FNVAlgorithms = "FNV32" | "FNV32A" | "FNV64" | "FNV64A";
 export type DigestAlgorithmName = WasmDigestAlgorithm | FNVAlgorithms;
 
+/*
+ * The largest digest length the current WASM implementation can support. This
+ * is the value of `isize::MAX` on 32-bit platforms like WASM, which is the
+ * maximum allowed capacity of a Rust `Vec`.
+ */
+const maximumDigestLength = 0x7FFF_FFFF;
```

Comment:
```suggestion
const MAX_32BIT_INTEGER = 0x7FFF_FFFF;
```
Nit: To match `Number.MAX_SAFE_INTEGER`

Reply:
Changed to uppercase, but didn't rename yet per above comment.